templates/connect-inject-authmethod-clusterrolebinding.yaml

```
@@ -0,0 +1,39 @@
+{{- if and (not .Values.connectInject.certs.secretName) (or (and (ne (.Values.connectInject.enabled | toString) "-") .Values.connectInject.enabled) (and (eq (.Values.connectInject.enabled | toString) "-") .Values.global.enabled)) }}
+{{- if .Values.global.bootstrapACLs }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

```suggestion
apiVersion: rbac.authorization.k8s.io/v1
```
I don't think the beta1 suffix is required anymore.

---

Turns out there was a mix in all of the rbac files. I've removed all of the beta tags across multiple files.